tests/integration/linux_bridge_test.py

```diff
@@ -116,14 +117,15 @@ def test_add_port_to_existing_bridge(bridge0_with_port0, port1_up):
     assertlib.assert_state(desired_state)
 
 
-def test_linux_bridge_uses_the_port_mac(port0_up, bridge0_with_port0):
-    port0_name = port0_up[Interface.KEY][0][Interface.NAME]
-    prev_port_mac = port0_up[Interface.KEY][0][Interface.MAC]
-    current_state = show_only((TEST_BRIDGE0, port0_name))
-    curr_iface0_mac = current_state[Interface.KEY][0][Interface.MAC]
-    curr_iface1_mac = current_state[Interface.KEY][1][Interface.MAC]
+def test_linux_bridge_uses_the_port_mac(eth1_up, bridge0_with_port0, port0_up):
+    current_state = show_only((TEST_BRIDGE0,
+                               port0_up[Interface.KEY][0][Interface.NAME]))
+    current_state_iface_macs = get_all_macs(current_state)
```

Comment:
If you use `macs` as mentioned above, you can then use `macs += get_all_macs(current_state)`.